.github/workflows/build.yml

```diff
@@ -29,6 +29,9 @@ jobs:
       - uses: sap/ai-sdk-js/.github/actions/setup@main
       - name: REUSE Compliance Check
         uses: fsfe/reuse-action@v5
+      - run: pnpm generate
+      - name: Enforce generated code is up to date
+        run: git status --porcelain
```

Comment:
[req] Shouldn't we check if there is any output from this command? The command actually always succeeds or?

Reply:
after experimenting a bit more, checking the output of `--porcelain` to not be empty is the only way I got to work reliably.